packages/react/src/utils.ts

```diff
@@ -327,9 +327,10 @@ export const disambiguateActionVariables = (
 
   if (action.acceptsModelInput || action.hasCreateOrUpdateEffect) {
     if (
-      action.modelApiIdentifier in variables &&
-      typeof variables[action.modelApiIdentifier] === "object" &&
-      variables[action.modelApiIdentifier] !== null
+      (action.modelApiIdentifier in variables &&
+        typeof variables[action.modelApiIdentifier] === "object" &&
+        variables[action.modelApiIdentifier] !== null) ||
+      !action.variables[action.modelApiIdentifier]
```

Comment:
is there a test we can add for this?

Reply:
I was thinking we could add a test post release of the email pw feature since those are the only triggers that currently have params that aren't either model input or exported params. We would need to 🎩 this with the trigger though

Reply:
:+1:

Reply:
Now that I'm thinking about it a bit more, maybe it's better to include a test for this now. @jcao49 maybe we can add the signUp trigger to one of the existing app's that we have a client for and use that as a test case